docs/workflow/concepts-key-concepts.mdx

```diff
@@ -37,7 +37,7 @@ When a workflow is run, the workflow definition is read and executed by the **wo
 -   Geocortex Mobile and Geocortex Workflow Server use the .NET workflow runtime
 
 :::note Types of Workflows
-Another distinction between workflows is **client** and [**server workflows**](https://docs.geocortex.com/workflow/latest/help/Default.htm#wf5/help/server-workflows.htm%3FTocPath%3DServer%2520Workflows%7C_____0). Client workflows can be run in an interface that uses visual elements, such as Geocortex Web or Mobile, while server workflows can only be run on Geocortex Workflow Server (which has no visual interface).
+Workflows are classified as **client** and **server** workflows. Client workflows support visual elements such as forms and prompts, and can be run on platforms with a visual interface, such as Geocortex Web or Geocortex Mobile. Server workflows do not support forms, prompts, or other visual elements, and can only be run on Geocortex Workflow Server, which has no visual interface. However, server workflows have a number of unique capabilities, such as the ability to connect to a database or mail server.
```

Comment:
Could we add back in the link to the server workflow docs that got removed in this change?

```suggestion
Workflows are classified as **client** and [**server workflows**](https://docs.geocortex.com/workflow/latest/help/Default.htm#wf5/help/server-workflows.htm%3FTocPath%3DServer%2520Workflows%7C_____0). Client workflows support visual elements such as forms and prompts, and can be run on platforms with a visual interface, such as Geocortex Web or Geocortex Mobile. Server workflows do not support forms, prompts, or other visual elements, and can only be run on Geocortex Workflow Server, which has no visual interface. However, server workflows have a number of unique capabilities, such as the ability to connect to a database or mail server.
```

Reply:
[User Story 40266: [Dev Center] [Workshop] Rephrase "Types of Workflows" note](https://dev.azure.com/geocortex/Geocortex/_workitems/edit/40266) from user story, it didn't mention to add the link back.

Reply:
It didn't, but that was a mistake on my part! Could you add it back? You should just be able to do it by clicking "commit suggestion" above, right in the github interface.